package.json

```
@@ -95,8 +95,8 @@
     "*": "prettier --experimental-cli --ignore-unknown --write"
   },
   "dependencies": {
-    "@docsearch/css": "^3.9.0",
-    "@docsearch/js": "^3.9.0",
+    "@docsearch/css": "beta",
+    "@docsearch/js": "beta",
```

```suggestion
    "@docsearch/css": "^4.0.0-beta",
    "@docsearch/js": "^4.0.0-beta",
```

Better to use range than tag. I'll update them when you guys hit v4 rc/stable.

---

Thanks.